routes/web.php

```diff
@@ -19,6 +19,8 @@
   
   //user routes
   Route::get("/users/create/","UserController@create");
+  Route::get("auth/user","AuthenticateController@user");
+  Route::get("auth/refresh","AuthenticateController@user");
```

Comment:
Do you think that it's logically consistent that refreshing a token and getting the authenticated user is semantically the same action? Why not just a simple function that only refreshes the token?